test/controllers/actions/AuthActionSpec.scala

```diff
@@ -51,7 +51,8 @@ class AuthActionSpec extends SpecBase {
       privateBetaAllowListEnabled = false,
       updateCaseEnabled = false,
       privateCitizenEnabled = false,
-      otherItemEnabled = false
+      otherItemEnabled = false,
+      welshToggleEnabled = true
```

Comment:
do you want the welsh feature on by default or just switched on for certain tests?